crates/jstz_api/tests/wpt.rs

```diff
@@ -214,15 +214,15 @@ impl jstz_core::Api for TestHarnessReportApi {
             let value = context
                 .global_object()
                 .get(js_string!(name), context)
-                .expect(&format!("globalThis.{} is undefined", name));
+                .unwrap_or_else(|_| panic!("globalThis.{} is undefined", name));
```

Comment:
Is this the linter? @ryutamago

Reply:
I think it's `clippy fix`. I'll change it back.